src/main/resources/i18n/Messages_en.properties

```diff
@@ -173,6 +173,15 @@ notification.document.completedSectionVariableUpdated.email.body.2=You are recei
 notification.document.completedSectionVariableUpdated.email.buttonLabel=View Document
 notification.document.completedSectionVariableUpdated.email.subject=Variable edited in a "Completed" document section
 notification.document.completedSectionVariableUpdated.email.title=Variable edited in a "Completed" document section
+notification.documentsUpdate.email.body.1=Dear Terraware User,
+notification.documentsUpdate.email.body.2=We are writing to notify you that Terraformation recently updated its [End User License Agreement (EULA)], [Privacy Policy] and [Terms of Use]. These updates, which you can view via the links above, went into effect in the Fall of 2024. By continuing to use our software, you agree to our updated terms.
```

Comment:
Nit: "The links above" seems like an odd word choice considering some of the links might be on the same line as that phrase depending on the user's window width. (I know this text was part of the spec.)

Reply:
@alexTerraformation what do you think about changing this from "view via the links above" to "view via these links" or "view via the aforementioned links"?

Reply:
I think either of those alternatives is fine.  @constanzauanini